partner_communication_switzerland/data/end_year_communication.xml

```diff
@@ -0,0 +1,52 @@
+  <odoo>
+    <data>
+        <!-- New picture e-mail -->
+        <record id="email_newyear" model="mail.template">
+            <field name="name">New Year</field>
+            <field name="model_id" ref="partner_communication.model_partner_communication_job"/>
+            <field name="email_from">compassion@compassion.ch</field>
+            <field name="reply_to">info@compassion.ch</field>
+            <field name="partner_to">${object.partner_id and object.partner_id.email and object.partner_id.id or False }</field>
+            <field name="subject">New Year Letter</field>
+            <field name="body_html" type="html">
+                <div>
+                    % set partner = object.partner_id
+                    % set sponsorships = object.get_objects()
+                    % set children = sponsorships.mapped('child_id')
+                    % set firstname = children.get_list('firstname', 3, 'many children')
+                </div>
+                <p>
+                    ${object.partner_id.salutation}
+                    <br/>
+                    First of all, we want to thank you warmly for your precious commitment towards ${firstname}. Through your support you are giving ${children.get('he')} hope and a future!
+                    <br/>
+                    Attached to this letter you will find the following documents:
+                    <br/>
+                    % if partner.payment_mode_id == 10:
+                    <br/>
+                    - Payment slips for the payment of your sponsorship for 2018 including the payment slips for the gifts (birthday gift, general gift and family gift)
+                    <br/>
+                    % endif
+                    % if partner.payment_mode_id != 10:
+                    <br/>
+                    - Payment slips for making gifts to your sponsored child (birthday gift, general gift and family gift)
+                    <br/>
+                    % endif
+                    % if partner != sponsorships.correspondent_id and partner.payment_mode_id == 10:
```

Comment:
Partner != sponsorships.mapped('correspondant_id') and partner.payment_mode_id.id == 10